### PR TITLE
different home

### PR DESCRIPTION
**Pull request by viggi-v**
does it look cool? if so we can make menu and other stuff based on this.
small improvement needed would be to render the text within the smoke so that we get the feeling of the engi logo appearing out of the smoke.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WebGL Smoke</title>
```
Comment (sbshah97) [inline index.html:7]: Maybe change the title as well :p

File: license.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (sbshah97) [inline license.txt:1]: Not including the LICENSE would be great :stuck_out_tongue_closed_eyes:

**Review by sbshah97 — CHANGES_REQUESTED** [review]
Remove unnecessary files! Apart from that looks great!

File: license.txt (hunk 2 above)
Comment (viggi-v) [reply license.txt:1]: yeah I just copied from codepen, so technically you are supposed to carry the license file.. I will remove the readme and the title.

Comment (sbshah97) [reply license.txt:1]: Cool works well for us atleast

**Review by sbshah97 — APPROVED** [review]
Good work for now!

**Pull request merged**